src/traits.rs

```diff
@@ -570,6 +570,20 @@ pub trait CpuExt: Debug {
     /// }
     /// ```
     fn frequency(&self) -> u64;
+
+    /// Returns the CPU's governor.
+    ///
+    /// ```no_run
+    /// use sysinfo::{CpuExt, GovernorKind, System, SystemExt, RefreshKind, CpuRefreshKind};
+    ///
+    /// let s = System::new_with_specifics(
+    ///     RefreshKind::new().with_cpu(CpuRefreshKind::everything()),
+    /// );
+    /// for cpu in s.cpus() {
+    ///     println!("{}", cpu.governor());
+    /// }
+    /// ```
+    fn governor(&self) -> GovernorKind;
```

Comment:
It should return an `Option` in case this information hasn't been retrieved yet.